src/components/Menu.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Disclosure, Transition } from '@headlessui/react'
+
+
+const Menu = (): JSX.Element =>  {
```

Comment:
Have this return a `ReactElement` rather than a `JSX.Element` (I realize I told you to put `JSX.Element` but I actually recently learned from Ally that `ReactElement` may be better since it's more type-specific than `JSX`)